pkgs/top-level/all-packages.nix

```diff
@@ -14383,6 +14383,8 @@ in
     inherit (darwin.apple_sdk.frameworks) SystemConfiguration;
   };
 
+  fusiondirectory = callPackage ../servers/web-apps/fusiondirectory { };
+
   fusionInventory = callPackage ../servers/monitoring/fusion-inventory { };
```

Comment:
Should we maybe keep the same formatting structure as ``fusionInventory``?

Reply:
What do you mean? FusionDirectory is not a monitoring tool, it's a webapp for managing LDAP trees.

Reply:
Sorry, thought they are related and I thought it would make sense to have the same naming structure (eg. ```fusionDirectory```).